sdk/javascript/src/utils/converter.js

```diff
@@ -136,7 +175,10 @@ const converter = {
 			}
 		};
 
-		return new (mapping[isSigned][size])(input.buffer, 0, 1)[0];
+		const DataType = mapping[isSigned][byteSize];
+		const typedBuffer = new DataType(1);
+		typedBuffer[0] = value;
```

Comment:
why this ?

Reply:
for readability, wasn't sure between this and alternative: `new DataType([value])`;

Reply:
i think alternative would be clearer